docs/installation/docker-prereq.md

```diff
@@ -0,0 +1,47 @@
+# Prerequisites
+
+## Requirements for using ISLE with Docker Compose
+
+- Docker 19.x+
+- Docker Compose version 1.25.x+
+- GNU Make 4.0+
+- Git 2.0+
+- [ISLE Docker Compose](https://github.com/islandora-devops/isle-dc)
+- At least 8GB of RAM (ideally 16GB)
+- An administrator account
```

Comment:
Administrator account is presumably on the host machine, but it would be useful to state that explicitly.